lib/player_manager.rb

```diff
@@ -0,0 +1,40 @@
+require 'ai_basic'
+require 'ai_minimax'
+require 'human_player'
+
+class PlayerManager
```

Comment:
So, it is not being used yet. It will be injected into the TerminalTicTacToe class and then passed along to the GameLoop class. I suppose I should include those changes in the PR as well..